tests/test_manaus/test_acm.py

```diff
@@ -218,7 +218,9 @@ def test_get_certificates(monkeypatch):
     assert len(certificates_net) == 1
     assert certificates_net[0].arn == 'arn:aws:acm:eu-west-1:cert2'
 
-    m_client.describe_certificate.side_effect = [{'Certificate': CERT3}]
+    # TODO :: NOT MEANT TO BE MERGED!!! JUST HERE TO FIX THE TEST IN PYTHON 3.7
```

Comment:
Do not forget this ;-)

Reply:
Turns out that this was actually the best way to fix the test, so I removed the comment